homeassistant/generated/zeroconf.py

```diff
@@ -53,6 +53,9 @@
     "_spotify-connect._tcp.local.": [
         "spotify"
     ],
+    "_ssh._tcp.local.": [
```

Comment:
We cannot use this discovery key, as this would mean any SSH server in the network would be discovered by the Smappee integration (which would be a lot of instances in my personal case).